src/patternfly/components/Tooltip/examples/tooltip-example.hbs

```diff
@@ -1,31 +1,31 @@
-{{#> tooltip direction="pf-m-right"}}
+{{#> tooltip direction="pf-m-right" tooltipContentId="right-tooltip-content"}}
     Hello!
     {{#*inline "tooltip-parent"}}
-        {{#> button btnClass='pf-m-primary' btnAttributes="aria-describedby=\"tooltip-description\" "}}
+        {{#> button btnClass='pf-m-primary' btnAttributes="aria-describedby=\"right-tooltip-content\" "}}
```

Comment:
I'd suggest in each instance of btnAttributes here to use the format `btnAttributes='aria-describedby="right-tooltip-content"'` to be consistent with other examples, and to avoid escaping characters where possible.